spec/factory_bot/internal_spec.rb

```diff
@@ -48,4 +58,28 @@
       FactoryBot::Internal.rewind_sequences
     end
   end
+
+  describe ".register_factory" do
```

Comment:
As mentioned in https://github.com/thoughtbot/factory_bot/pull/1291#discussion_r283066959, our style guide (and also I 😄 ) would prefer some newlines in these tests.